wagtail/snippets/models.py

```diff
@@ -120,8 +121,24 @@ def register_deferred_snippets():
         _register_snippet_immediately(registerable, viewset)
 
 
-def create_extra_permissions(*args, using=DEFAULT_DB_ALIAS, **kwargs):
-    model_cts = ContentType.objects.get_for_models(
+def create_extra_permissions(
+    app_config, *args, using=DEFAULT_DB_ALIAS, apps=global_apps, **kwargs
+):
+    if not app_config.models_module:
+        return
```

Comment:
I don't think we need this check. Django's create_permissions handler does this because it's configured to run for every app (the `post_migrate.connect` call does not specify a `sender`), and some apps won't have a models module. In our case, we pass `sender=self` so that we're only triggering this on the wagtailsnippets app in particular - we know that wagtailsnippets has a models module, and in any case we're not doing anything that relies on it (we're not calling `app_config.get_models()`).